src/field/pokemon.ts

```diff
@@ -1390,7 +1391,11 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
 
   abstract getBossSegmentIndex(): number;
 
-  getMoveset(bypassSummonData: boolean = false): PokemonMove[] {
+  /**
+   * @param bypassSummonData - Whether to get the Pokemon's actual moveset
```

Comment:
```suggestion
   * @param bypassSummonData - Whether to get the unmodified Pokemon's moveset. Default: `false`
```
not sure what the best way to express this is. But I feel that "actual" could be interpreted either way

Reply:
Good point, I'll add some clarification to the doc.

Reply:
What do you think of this?

> (Default `true`) Whether to get the Pokemon's actual/unmodified moveset (`true`) or the Pokemon's temporary/modified moveset (such as due to Transform) (`false`).